src/Controller/ArticlesController.php

```diff
@@ -855,6 +856,17 @@ private function articlePageArguments(Request $request, string $id, int $version
                     $infoBars[] = new InfoBar('This research is available in a <a href="'.$rdsArticles[$item->getId()].'">reproducible view</a>.', InfoBar::TYPE_WARNING);
                 }
 
+                $dismissibleInfoBars = $this->getParameter('dismissible_info_bars');
+                foreach ($dismissibleInfoBars as $infoBarName => $infoBarConfiguration) {
+                    if (in_array($item->getId(), $infoBarConfiguration['article_ids'])) {
+                        if ($item instanceof ArticlePoA) {
+                            continue;
+                        }
+                        // TODO: pass in `cookieDuration` or `cookieExpires` as constants
+                        $infoBars[] = new InfoBar($infoBarConfiguration['text'], InfoBar::TYPE_DISMISSIBLE, $infoBarName, new DateTimeImmutable('+365 days'));
```

Comment:
how configurable should 365 days (or any other value) be?
- constant in this `ArticlesController`
- constant for all dismissible InfoBars in `journal` (there's only this one anyway)
- `parameters.yml` so that it can change by environment in theory (e.g. 1 minute locally, 1 year in `prod`)
- `parameters.yml` but configurable for each InfoBar (probably crazy)